components/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -12,11 +12,21 @@ import HomeIcon from 'icons/HomeIcon'
 import BlockIcon from 'icons/BlockIcon'
 import NextLink from 'next/link'
 
+export interface BreadCrumbQueryType {
+  id?: number | string | null | undefined
+  hash?: string | null | undefined
+}
+
+interface BreadCrumbToType {
+  pathname?: RoutePaths | string | null | undefined
+  query?: string | null | BreadCrumbQueryType | undefined
+}
+
 interface BaseBreadcrumbLinkProps {
   icon: ReactNode
   label: ReactNode
   isCurrent?: boolean
-  to?: string
+  to?: BreadCrumbToType | string
```

Comment:
May be better to use `Url` or `UrlObject` instead of custom type? See how implement `Link` in `next/link`